static/bidder-info/adf.yaml

```diff
@@ -23,5 +23,5 @@ capabilities:
       - video
 userSync:
   redirect:
-    url: "https://cm.adform.net/cookie?redirect_url={{.RedirectURL}}"
+    url: "https://c1.adform.net/cookie?redirect_url={{.RedirectURL}}"

```

Comment:
@braizhas redirect url isn't redirecting

`https://c1.adform.net/cookie?redirect_url=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dadf%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%22%24UID%22`

Reply:
Browser should already have Adform cookie for redirect to happen. Open this link second time and you will be redirected.

Reply:
Able to redirect at second instance.

Reply:
@braizhas If the adform cookie is not present in the browser then the expected behaviour for adapter is to create new UID.

Reply:
Why? There is no guarantee that browser will accept new UID cookie (safari, ff). It's a part of business logic on how match pairs are created.

Reply:
@braizhas This is expected behaviour. 
If you would like to discuss this more, let's use the prebid server PMC meeting that happens on wednesday.

Reply:
I don't see how such logic conflicts with prebid-server ID sync. Could you send me an invitation to the meeting?